test/unit_tests/modules/classes/BreakpointHandler-spec.js

```diff
@@ -10,8 +10,8 @@ var BASE_LOC = '../../../../cfgov/unprocessed/js/';
 
 beforeEach( function() {
   args = {
-    enter:      function() {},
-    leave:      function() {},
+    enter:      function() { return; },
```

Comment:
Why is the return statement necessary? Are we requiring that all functions ( not constructors ) have a return statement?